app/models/sub_service_request.rb

```diff
@@ -320,7 +320,7 @@ def is_locked?
   def can_be_edited?
     if organization.has_editable_statuses?
       self_or_parent_id = find_editable_id(self.organization.id)
-      EDITABLE_STATUSES[self_or_parent_id].include?(self.status)
+      EDITABLE_STATUSES[self_or_parent_id].include?(self.status) && !is_complete?
```

Comment:
@jayhardee9 
According to Wenjun, even if it is an editable status, the users should not be able to edit a Complete SSR, no matter what. She said that instead, it means that users should be able to draft a new SSR for that organization.